tests/unit/modules/test_virt.py

```diff
@@ -3558,7 +3558,7 @@ def test_capabilities(self):
       <cpuselection/>
       <deviceboot/>
       <disksnapshot default='on' toggle='no'/>
-      <acpi default='on' toggle='yes'/>
```

Comment:
Do we only want to test the case where this is True or should we test with both True and False?